content/posts/August-2020.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "August 2020"
+date: 2020-08-13
+draft: false
+---
+
+## Week 32 (August 3rd - August 7th) and 33 (August 10th - August 13th)
+
+- Last week, we renamed our GitHub organisation to `packit`.
```

Comment:
'Last week' is relative
```suggestion
-We renamed our GitHub organisation to `packit`.
```

Reply:
Sorry for editing your comment to contain the space.